package-lock.json

```diff
@@ -1,28 +1,46 @@
 {
   "name": "express-middleware",
-  "version": "1.9.5",
+  "version": "2.0.0",
```

Comment:
You should completely remove this file as npm coming with node 6 will not update it

Reply:
It's usefull for newest version of node

Reply:
There will be issues, we use node 6 locally and when installing a dependency this file will become out of date and lead to unexpected dependencies versions installed in clients